Dockerfile

```diff
@@ -40,3 +45,19 @@ RUN SECRET_KEY=none django-admin compress
 
 # Run application
 CMD gunicorn wagtailio.wsgi:application
+
+
+FROM backend AS dev
+
+# Install Heroku CLI
+RUN curl https://cli-assets.heroku.com/install-ubuntu.sh | sh
+
+# Install AWS CLI
+RUN curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "/tmp/awscli-bundle.zip"
```

Comment:
Would it be worth doing this in one `RUN` by chaining these with `&&` and deleting the temporary files at the end? Not sure how much of a problem bigger images will be in practice, but it might be a bit kinder to people's hard drives.

Reply:
Done, thanks!